services/autoscaling/src/simcore_service_autoscaling/utils/rabbitmq.py

```diff
@@ -68,44 +65,45 @@ async def post_task_log_message(app: FastAPI, task: Task, log: str, level: int)
 
 
 async def create_autoscaling_status_message(
-    docker_client: AutoscalingDocker,
     app_settings: ApplicationSettings,
     cluster: Cluster,
+    cluster_total_resources: Resources,
+    cluster_used_resources: Resources,
 ) -> RabbitAutoscalingStatusMessage:
     assert app_settings.AUTOSCALING_EC2_INSTANCES  # nosec
-    assert app_settings.AUTOSCALING_NODES_MONITORING  # nosec
-    monitored_nodes = [
-        i.node
-        for i in itertools.chain(
-            cluster.active_nodes, cluster.drained_nodes, cluster.reserve_drained_nodes
-        )
-    ]
-    (total_resources, used_resources) = await asyncio.gather(
-        *(
-            utils_docker.compute_cluster_total_resources(monitored_nodes),
-            utils_docker.compute_cluster_used_resources(docker_client, monitored_nodes),
-        )
-    )
+
+    origin = "unknown"
```

Comment:
what about constant in case you want to check this case somwhere else? e.g.
```python
# some_other_module.py
from ...utils.rabbitmp import UNKNOWN

if origin == UNKNOWN:
   ...

```

thought: Sometimes it is worth having a proper sentinel that just `None`. Specially when it has many possible interpretations. I  wonder whether we could define some sort of base class for all these sentinels so it puts a bit of "order" . Something like

```python
class Sentinel:
    def __init__(self, name, description):
        self.name = name
        self.description = description
        
    def __equal__(self, other) -> bool :
         return self.name == other.name

    def __str__(self):
        return f"{self.name}: {self.description}"

UNKNOWN =  Sentinel("unknown", "When a value cannot be determined")
UNSET = Sentinel("unset", "The value was never set")


def foo(value: int | Sentinel = UNSET):
    ...
def bar(value: SomeClass | Sentinel = UNKNOWN):
    ...
```

or simply an enum to gather all of them as a it would be a vocabulary?

```python
class Sentinel(Enum):
     UNKNOWN = auto()
     UNSET = auto()
```

Reply:
I am not quite sure I get the point here. but I'll try to improve it in the next PR